x-pack/plugins/cases/public/components/all_cases/columns.tsx

```diff
@@ -246,19 +244,15 @@ export const useCasesColumns = ({
       },
       truncateText: true,
     },
-    ...(!disableAlerts
-      ? [
-          {
-            align: RIGHT_ALIGNMENT,
-            field: 'totalAlerts',
-            name: ALERTS,
-            render: (totalAlerts: Case['totalAlerts']) =>
-              totalAlerts != null
-                ? renderStringField(`${totalAlerts}`, `case-table-column-alertsCount`)
-                : getEmptyTagValue(),
-          },
-        ]
-      : []),
+    {
```

Comment:
don't we need to check the features here?

Reply:
I talked with the Observability team and they want the count of the alerts even when the `syncAlerts` is disabled. Sorry, I forgot to mention it.